verification/src/compiler/fetcher.rs

```diff
@@ -72,21 +70,26 @@ pub async fn write_executable(
     let save_result = {
         let file = file.clone();
         let data = data.clone();
-        tokio::task::spawn_blocking(move || -> Result<(), FetchError> {
-            std::fs::create_dir_all(&folder)?;
-            std::fs::remove_file(file.as_path()).or_else(|e| {
-                if e.kind() == ErrorKind::NotFound {
-                    Ok(())
-                } else {
-                    Err(e)
-                }
-            })?;
-            let mut file = create_executable(file.as_path())?;
-            std::io::copy(&mut data.as_ref(), &mut file)?;
-            Ok(())
+        let span = tracing::debug_span!("save executable");
+        tokio::task::spawn_blocking(move || {
+            span.in_scope(|| {
```

Comment:
I know you've talked about this, but what's the difference between `in_scope` and `enter`?
I mean, in the code it's just the same
https://docs.rs/tracing/latest/src/tracing/span.rs.html#785-788
https://docs.rs/tracing/latest/src/tracing/span.rs.html#1100-1103

Reply:
changed to `.enter()`